subjects/devops/find-files/README.md

```diff
@@ -0,0 +1,49 @@
+## find-files
+
+### Instructions
+
+"start finding ..."
+
+Create a file `find-files.sh`, which will look for and show, in the current directory and its sub-folders:
+
+everything that starts with an `a` or,
+all the files ending with a `z` or,
```

Comment:
There is an extra `or` at the end of this line. I would finish the line with a `.`